### PR TITLE
Add ORCID import option to PURE page

### PR DESCRIPTION
**Pull request by pherterich**
Just before leaving Birmingham, I managed to grab a few screenshots to explain how to add code from ORCID if it has a DOI from zenodo/figshare. I can work on the wording but I won't have access to PURE beyond tomorrow to redo or add any screenshots.

Hope this is still helpful.

### CODE REVIEW

File: pure.md [hunk 1]
```diff
@@ -89,3 +89,29 @@ Add details of any **Relations** the record has to any existing records in Pure
...
 Finally, set the **Visibility** according to how you wish the record to be available
 and press **Save**.
+
+
+## How to import code to PURE via ORCID
+
+This guide has been written with reference to the Pure instance at the
+University of Birmingham, which is version 5.14.1 at the time of writing.
+
+*Prerequisits:*
```
Comment (olexandr-konovalov) [inline pure.md]: Prerequisits - > Prerequisites

**Review by olexandr-konovalov — APPROVED** [review]
Thank you very much @pherterich - this is very useful! I can't test it in our Pure installation since it does not have ORCID as an enabled import source for PURE (I will try to find out why and suggest to add it). Pinging @pgmccann who has more insights into the internals of our Pure installation.

File: pure.md (hunk 1 above)
Comment (pherterich) [reply pure.md]: Thanks, I knew there would be typos in there that I didn't spot! Fixed now

**Comment by pherterich** [comment]
I'm lucky enough to have admin rights here in Birmingham, anyone with those rights can enable that. I think some institutions might be a bit hesitant because the metadata quality isn't great but it's really useful for more unusual outputs I find.

**Comment by pgmccann** [comment]
We have 5.13 at the moment, which doesn't support this. We're planning to go to 5.15 soon, so hopefully it will then be enabled once there's been a chance to do some testing.

**Pull request merged**